### PR TITLE
Remove old co2-tracker-utils dependency

### PR DESCRIPTION
**Pull request by Lothiraldan**

### CODE REVIEW

File: meta.yaml [hunk 1]
```diff
@@ -19,7 +19,6 @@ requirements:
   run:
     - python>3.6
     - apscheduler
-    - co2-tracker-utils
```
Comment (kngoyal) [inline meta.yaml:22]: remove from setup.py too ?

Comment (Lothiraldan) [reply meta.yaml:22]: Good catch!

File: setup.py [hunk 2]
```diff
@@ -5,14 +5,14 @@
...
     "APScheduler",
-    "co2-tracker-utils",
     "dash",
     "dash_bootstrap_components",
     "dataclasses",
     "fire",
     "pandas",
     "pynvml",
     "requests",
+    "typing>=3.7.4; python_version<'3.5'",
```
Comment (kngoyal) [inline setup.py:15]: why python version<3.5 ?
In the docs, the recommended version is 3.6

Comment (Lothiraldan) [reply setup.py:15]: Ok good catch, let's remove it

**Review by kngoyal — APPROVED** [review]

**Pull request merged**